README.md

```diff
@@ -166,15 +166,25 @@ mode or the command line mode.
 
 There are many bugs and TODOs.
 
-## Installation
+Plotman will always look in the current directory for the `config.yaml` file
+([see also](https://github.com/ericaltendorf/plotman/pull/61#issuecomment-812967363)).
 
-This program requires `psutil`, `pyfakefs`, `pyyaml` and `texttable`.
+## Installation
 
 Installation for Ubuntu 20.04:
 
-```
-# Install Python package system
-sudo apt-get install python3-pip
+1. Plotman assumes that a functioning [Chia](https://github.com/Chia-Network/chia-blockchain)
+   installation is present on the system. Activate your `chia` environment by typing
+   `. /path/to/your/chia/install/activate`.
+2. Then, clone the plotman repository (*note that these instructions are currently only
+   valid for the development branch*):
 
-python3 -m pip install psutil pyfakefs pyyaml texttable
-```
+       git clone https://github.com/ericaltendorf/plotman.git -b development
+3. Enter the Plotman directory
+
+       cd plotman
+4. Install plotman using pip:
+
+       pip install .
```

Comment:
I don't know what workflow we want to encourage in the interrim here until plotman is on PyPI.  We could skip the git cloning and just `pip install git+https://github.com/ericaltendorf/plotman@development` or such.

Reply:
Fine with me, definitely easier with fewer steps and no explicit git knowledge required! Haven't tested it but it might need a `#egg=plotman` at the end of the URL? At least I've seen that construct in other similar cases before.

Reply:
Ok, I reworked this in commit 8b06c62, please check if this is resolved.

Reply:
Seems good, thanks.